### ISSUE TITLE
Testing of first revision

### ISSUE THREAD

**Issue by henrikssn** [maribu/at86rf233-breakout#1]
Did you ever end up testing this board?

**Comment by maribu** [comment]
No, after testing the AT86RF215 breakout, we decided to just go with that. I do have some PCBs and the required parts to populate it lying around somewhere.

I will try to solder it soonish, if I manage to find the stuff and some spare time.
